apps/docs/pages/guides/auth/server-side/email-based-auth-with-pkce-flow-for-ssr-with-third-party.mdx

```diff
@@ -0,0 +1,332 @@
+import Layout from '~/layouts/DefaultGuideLayout'
+
+export const meta = {
+  title: 'Email Auth with PKCE flow for SSR using a third-party email provider',
+  description:
+    'Learn how to configure email authentication in your server-side rendering (SSR) application to work with the PKCE flow using a third-party email provider.',
+  subtitle:
+    'Learn how to configure email authentication in your server-side rendering (SSR) application to work with the PKCE flow using a third-party email provider.',
+}
+
+### Setting up SSR client
```

Comment:
maybe we can even take out all this bit at the front (no need to repeat it if we can just link it) and have this page entirely about the code for generating links

Reply:
Makes sense. I have removed the duplicate part from the front. Is this what you meant?

Reply:
Yep exactly! Thanks!